pkg/cmd/server/api/types.go

```diff
@@ -83,6 +95,8 @@ const (
 	ControllersAll = "*"
 )
 
+type FeatureSet map[OpenShiftFeature]util.Empty
```

Comment:
Is this just to support a `Has`?  I think I'm ok dealing with `[]string` over the cruft that this adds.  If you want to try to keep it, I'm willing to take another look after the int->string and string->int gets refactored into just string, but right now my inclination is "not worth the pain".